src/app/containers/PageHandlers/withContexts.test.jsx

```diff
@@ -41,7 +41,7 @@ describe('withContexts HOC', () => {
     data: { status: 200 },
   };
 
-  shouldShallowMatchSnapshot(
+  shouldMatchSnapshot(
```

Comment:
Hmmm this test now doesn't do the same thing 🙈 You could replace `<ContextsHOC {...props} />` with `<Component />` and it would be the same.

Can you update `Component` to use the `useContext` hook for every context, and then render it in some way? Maybe JSON.stringify it? That way if a context in `WithContexts` gets removed, the snapshot will change? Happy to discuss this on slack :)